### PR TITLE
Clarify startup behaviour for performance counters

### PR DESCRIPTION
**Pull request by kbaley**
- Clarified that endpoints must be restarted before data will be collected
- Added instructions to install counters

### CODE REVIEW

**Comment by indualagarsamy** [comment]
@kbaley - can you fix the build error regarding language?

 ----> ErrorsException : Invalid word(s) detected. (Invalid word(s) may be in a snippet(s).) The following strings are not allowed: you, we, our, your, us, please, yourself.
Invalid word detected on line 35: ' you '
    line: NOTE: Regardless of whether you enable the SLA and Critical Time counters, you must re-start your endpoints after installing the counters before data will start to be collected.
Invalid word detected on line 35: ' your '
    line: NOTE: Regardless of whether you enable the SLA and Critical Time counters, you must re-start your endpoints after installing the counters before data will start to be collected.

**Comment by kbaley** [comment]
@indualagarsamy Done

File: samples/performance-counters/sample.md [hunk 1]
```diff
@@ -37,12 +37,14 @@ The following will result
...
 ## Enabling Counters For The Endpoint
 
-Both the SLA and Critical time are enabled and configured in configuration code.
+Both the SLA and Critical Time counters are enabled and configured in configuration code.
 
 snippet:enable-counters
 
 The other counters are enabled by default.
 
+NOTE: Regardless of whether the SLA and Critical Time counters are enabled, all endpoints must be restarted after installing the counters before data will start to be collected.
```
Comment (indualagarsamy) [inline samples/performance-counters/sample.md]: How about:

NOTE: Regardless of whether the SLA and Critical Time counters are enabled, all endpoints must be restarted after installing the counters in order to start collecting the new data.

File: nservicebus/operations/performance-counters.md [hunk 2]
```diff
@@ -113,7 +113,20 @@ Enabled by default and will only write to the counter if it exists.
...
+Install-NServiceBusPerformanceCounters
+```
+
+To list the installed counters use
+
+```ps
+Get-Counter -ListSet NServiceBus | Select-Object -ExpandProperty Counter
+```
+
+NOTE: After installing the performance counters, all endpoints must be restarted before data will start to be collected.
```
Comment (indualagarsamy) [inline nservicebus/operations/performance-counters.md]: How about:
NOTE: After installing the performance counters, all endpoints must be restarted in order to start collecting the new data.

**Comment by kbaley** [comment]
@indualagarsamy Yeah, that sounds better. Fixed.

**Comment by indualagarsamy** [comment]
@kbaley - not seeing your commit after the note fix. :(

**Comment by kbaley** [comment]
@indualagarsamy They're there now. Can't think of any possible explanation for the error last time. It certainly wasn't because I forgot to do a `git add .` if that's what you're thinking.

**Pull request merged**